src/gh_pr/utils/export.py

```
@@ -326,8 +326,43 @@ def export_batch_report(
+                "results": results,
+                "exported_at": datetime.now().isoformat(),
+            }
+
+            timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+            output_filename = filename or f"batch_report_{timestamp}.json"
+            output_path = Path(output_filename)
+            output_path.write_text(json.dumps(export_data, indent=2, default=str))
+            return str(output_path)
+
+        else:
+            raise ValueError(f"Unsupported format: {format}")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)